en/docs/deploy/docker.md

```diff
@@ -0,0 +1,94 @@
+			
+
+    5.  Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/) **version-nginx-0.22.0**. You can get the raw file for the recommended version [here](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.22.0).
+
+    6.  Add the WSO2 Helm chart repository.
+
+        ```java
+        helm repo add wso2 https://helm.wso2.com && helm repo update
+		```
+
+Once you have kubernetes running on docker, follow the steps given below to deploy WSO2 Identity Server in your local environment. 
+
+!!! note ""
+	-	The local copy of the `wso2/kubernetes-is` git repository will be referred to as `KUBERNETES_HOME`.
+	-	`<KUBERNETES_HOME>/advanced/` will be referred to as `HELM_HOME`.
+
+### Step 1 - Clone the Kubernetes Resources for WSO2 Identity Server Git repository
+
+```java
+git clone https://github.com/wso2/kubernetes-is.git
+```
+
+### Step 2 - Change the configurations as required 
+
+1.	The default configurations work well for the basic deployment of the product. However, if there is anything specific that needs to be configured, change the respective files in `<HELM_HOME>/is-pattern-1/`. 
+
+2.	 Open `<HELM_HOME>/is-pattern-1/values.yaml` and provide the values as mentioned in the second step **(Provide configurations)** of the **Quick Start Guide** [here](https://hub.helm.sh/charts/wso2/is-pattern-1).
+
+### Step 3 - Deploy WSO2 Identity Server
+
+Execute the following command to deploy the product. Here, `NAMESPACE` is the Kubernetes Namespace in which the resources are deployed and the `<RELEASE_NAME>` can be any name that you choose for the deployed instance. 
+
+```java
+helm install --dep-up --name <RELEASE_NAME> <HELM_HOME>/is-pattern-1 --namespace <NAMESPACE>
+```
+
+### Step 4 - Access the management console
+
+To access the console in the environment,
+
+1.	Obtain the external IP of the Ingress resources by listing down the Kubernetes Ingresses as shown below. 
+
+	```java
+	kubectl get ing -n <NAMESPACE>
+	```
+	This gives you the `<RELEASE_NAME>` and `<EXTERNAL-IP>` as shown below. 
+
+	```java 
+	NAME                       HOSTS                  ADDRESS        PORTS     AGE
+	wso2is-ingress             <RELEASE_NAME>         <EXTERNAL-IP>  80, 443   3m
+	```
+
+2.	Add the information obtained above in the /etc/hosts file as an entry. 
+
+	```java
+	<EXTERNAL-IP>	<RELEASE_NAME>
+	```
+
+3.	Try navigating to `https://<RELEASE_NAME>/carbon` from your favorite browser.
+
+### Try it Out 
+
+You can now test the functionalities of WSO2 Identity Server with your app. You can alternatively choose a sample app from [here](../../learn/deploying-the-sample-app) and follow the steps given to deploy the chosen application. 
```

Comment:
Alternatively, `[choose a sample application](insertlink)` and follow the steps....

With the new structure we won't have the 'deploying-the-sample-app' page. Instead you can link them to the samples overview page `(../../samples/overview)`